emacs/lsp-superbol.el

```diff
@@ -26,15 +26,16 @@
   :link '(url-link "https://github.com/OCamlPro/superbol-studio-oss")
   :package-version '(lsp-mode . "8.0.1"))
 
-(load (expand-file-name "lsp-superbol-customs.el"
-			(file-name-directory load-file-name)))
+;(load (expand-file-name "lsp-superbol-customs.el"
+;			(file-name-directory load-file-name)))
 
 ;; ---
 
 (defun lsp-superbol--server-command ()
   "Startup command for the Superbol LSP language server."
   ;; (list (lsp-package-path 'superbol-language-server) "lsp"))
-  (list (expand-file-name "superbol-free" lsp-superbol-path) "lsp"))
+  (message "lsp-superbol--server-command")
```

Comment:
Why pollute user's Messages buffer like this? One should at lease make those conditional.